dhall/src/Dhall/Parser/Expression.hs

```diff
@@ -759,7 +759,9 @@ parsers embedded = Parsers {..}
 
                         return (c, d) )
 
-                    m <- toMap ((a, b) : e)
+                    let combine _ = liftA2 (flip Combine)
```

Comment:
Why the `flip`?

Reply:
It's because `toMap` internally uses `Dhall.Map.fromListWithKey` which uses `Data.Map.fromListWithKey` which does things backwards